js/packages/teams-ai/package.json

```diff
@@ -25,17 +25,17 @@
     "main": "./lib/index.js",
     "types": "./lib/index.d.ts",
     "peerDependencies": {
-        "botbuilder": "^4.23.0",
-        "openai": "^4.61.0"
+        "botbuilder": "^4.23.1",
+        "openai": "^4.68.2"
```

Comment:
Do `openai` dependency in samples have to be updated to prevent dependency conflicts?

Reply:
good catch, thanks